mlrun/feature_store/common.py

```diff
@@ -145,6 +145,16 @@ def verify_feature_set_permissions(
     db.verify_authorization(auth_input)
 
 
+def verify_feature_set_exists(feature_set):
+    db = feature_set._get_run_db()
+    project, uri, tag, _ = parse_feature_set_uri(feature_set.uri)
+
+    try:
+        db.get_feature_set(feature_set.fullname, project, tag)
```

Comment:
Per Orit's comment, you have to verify that the featureset has a schema too:
https://jira.iguazeng.com/browse/ML-1698?focusedCommentId=87826&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-87826